quantum/pointing_device/pointing_device.c

```diff
@@ -377,28 +390,28 @@ void pointing_device_set_cpi_on_side(bool left, uint16_t cpi) {
 }
 
 /**
- * @brief clamps int16_t to int8_t
+ * @brief clamps int16_t to int8_t, or int32_t to int16_t
```

Comment:
Has been included in [a separate PR](https://github.com/qmk/qmk_firmware/pull/24422).